modules/common/html-utils/src/main/resources/xml/xslt/html5-outliner.xsl

```diff
@@ -67,7 +67,8 @@
                     <a href="{$relative-path}#{$id}">
                         <!-- TODO: try to not "depend" on the TTS namespace here -->
                         <xsl:copy-of select="$heading/ancestor-or-self::*/@tts:*"/>
-                        <xsl:copy-of select="$heading-content"/>
+                        <xsl:copy-of select="$heading-content[count(.|../@*)=count(../@*)]"/>
+                        <xsl:copy-of select="$heading-content[self::* | self::text()]"/>
```

Comment:
Hmm I fail to see what this does. Can you explain?

Reply:
Good question. I'm guessing the intent is to exclude some types of content. This part can probably be reverted.

Reply:
OK.